oshmem/mca/atomic/ucx/atomic_ucx_cswap.c

```diff
@@ -19,6 +19,23 @@
 
 #include "atomic_ucx.h"
 
+
+#define OSHM_ATOMIC_SET_INT(ptr, size, value, my_pe) do { \
```

Comment:
BTW, why do we need a macro? seems it's used only once

Reply:
I've put it in a macro because it was a single line before, setting a value of integer felt like it shouldn't take that much space inside that function when someone just try to understand what it does, also, it might be useable in the future for some different atomic operation that will be implemented in UCX?

Reply:
Re future, IMO sounds unlikely, as the API is quite feature-complete, @yosefe  WDYT?

Reply:
it will not be reused in UCX
can make it if {} .. else if {} .. else {}, it will be quite short.

Reply:
Ok, moved it back into the function